SaltyChat.Client/source/app.ts

```diff
@@ -455,17 +453,12 @@ export class SaltyVoice {
 
                 let muffleIntensity = null;
                 if (Config.enableMuffling) {
-                    let npRoomId = native.getRoomKeyFromEntity(nextPlayer.scriptID);
-
-                    if (localRoomId != npRoomId && !native.hasEntityClearLosToEntity(localScriptId, nextPlayer.scriptID, 17)) {
-                        muffleIntensity = 10;
-                    } else {
-                        let pVehicle = alt.Player.local.vehicle;
-                        let nVehicle = nextPlayer.vehicle;
-                        if (pVehicle != nVehicle) {
-                            if (pVehicle && !hasOpening(pVehicle)) muffleIntensity += 4;
-                            if (nVehicle && !hasOpening(nVehicle)) muffleIntensity += 4;
-                        }
+                    muffleIntensity = roomComparison(nextPlayer)
```

Comment:
Though the idea is fine, this should be just optional. Remember that there are custom mlos or even map replacements which could interfere with the ids you provided

Reply:
That is a good point. @Phill030 also said this could be a major problem. Maybe there is a change to put that as an optional configuration together with the RoomCombinations.